NewProjects.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
[nitpick] The file name convention followed in the repository is snake case with hyphens and not the camelcase.